cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
I'm going to guess that they might have been used for generation of some static documentation in format, which required some escaping. It shouldn't matter now and must be safe to remove slashes in docstring during further refactoring.